index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>My Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="toolbar">
+        <a>Welcome</a>
+        <span> <img src="images/toggle.png" alt="toggle icon" /> </span>
```

Comment:
- [ Optional] good job by changing the logo to a link but it would be better if you also if the link send the user to the home page